it/v1/endpoints/CreateForeignPropertyControllerISpec.scala

```diff
@@ -0,0 +1,645 @@
+      |      "other": 5000.99634874383248236385
+      |    }
+      |  },
+      |  "foreignProperty": [
+      |    {
+      |      "countryCode": "FRA",
+      |      "income": {
+      |        "rentIncome": {
+      |          "rentAmount": 5000.99634874383248236385,
+      |          "taxDeducted": 5000.99634874383248236385
+      |        },
+      |        "foreignTaxCreditRelief": false,
+      |        "premiumOfLeaseGrant": 5000.99634874383248236385,
+      |        "otherPropertyIncome": 5000.99634874383248236385,
+      |        "foreignTaxTakenOff": 5000.99634874383248236385,
+      |        "specialWithholdingTaxOrUKTaxPaid": 5000.99634874383248236385
+      |      },
+      |      "expenditure": {
+      |        "premisesRunningCosts": 5000.99634874383248236385,
+      |        "repairsAndMaintenance": 5000.99634874383248236385,
+      |        "financialCosts": 5000.99634874383248236385,
+      |        "professionalFees": 5000.99634874383248236385,
+      |        "costsOfServices": 5000.99634874383248236385,
+      |        "travelCosts": 5000.99634874383248236385,
+      |        "residentialFinancialCost": 5000.99634874383248236385,
+      |        "broughtFwdResidentialFinancialCost": 5000.99634874383248236385,
+      |        "other": 5000.99634874383248236385
+      |      }
+      |    }
+      |  ]
+      |}
+      |""".stripMargin)
+
+  val bothExpensesSuppliedRequestJson: JsValue = Json.parse(
+    """
+      |{
+      |  "fromDate": "2018-04-06",
+      |  "toDate": "2019-04-06",
+      |  "foreignFhlEea": {
+      |    "income": {
+      |      "rentAmount": 5000.99,
+      |      "taxDeducted": 5000.99
+      |    },
+      |    "expenditure": {
+      |      "premisesRunningCosts": 5000.99,
+      |      "repairsAndMaintenance": 5000.99,
+      |      "financialCosts": 5000.99,
+      |      "professionalFees": 5000.99,
+      |      "costsOfServices": 5000.99,
+      |      "travelCosts": 5000.99,
+      |      "other": 5000.99,
+      |      "consolidatedExpenses": 5000.99
+      |    }
+      |  },
+      |  "foreignProperty": [
+      |    {
+      |      "countryCode": "FRA",
+      |      "income": {
+      |        "rentIncome": {
+      |          "rentAmount": 5000.99,
+      |          "taxDeducted": 5000.99
+      |        },
+      |        "foreignTaxCreditRelief": false,
+      |        "premiumOfLeaseGrant": 5000.99,
+      |        "otherPropertyIncome": 5000.99,
+      |        "foreignTaxTakenOff": 5000.99,
+      |        "specialWithholdingTaxOrUKTaxPaid": 5000.99
+      |      },
+      |      "expenditure": {
+      |        "premisesRunningCosts": 5000.99,
+      |        "repairsAndMaintenance": 5000.99,
+      |        "financialCosts": 5000.99,
+      |        "professionalFees": 5000.99,
+      |        "costsOfServices": 5000.99,
+      |        "travelCosts": 5000.99,
+      |        "residentialFinancialCost": 5000.99,
+      |        "broughtFwdResidentialFinancialCost": 5000.99,
```

Comment:
Can you also put consolidatedExpenses in here? The paths below will also need updating